From d3687cb45f88245068027956b3e42f13f854dbc0 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Mon, 12 Sep 2016 21:00:01 +0300
Subject: [PATCH] configs: rcar-gen3-common: Enable hush parser

This enables hush parser, which supports
conditionals and can be used to make
U-Boot scripts more flexible.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 include/configs/rcar-gen3-common.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/rcar-gen3-common.h b/include/configs/rcar-gen3-common.h
index 2c2a5a5..aa692e5 100644
--- a/include/configs/rcar-gen3-common.h
+++ b/include/configs/rcar-gen3-common.h
@@ -41,6 +41,8 @@
 #define CONFIG_SYS_THUMB_BUILD
 #define CONFIG_SYS_GENERIC_BOARD
 
+#define CONFIG_SYS_HUSH_PARSER
+
 #define CONFIG_REMAKE_ELF
 
 /* boot option */
-- 
1.9.3

